<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySerialWidget</class>
 <widget class="QWidget" name="MySerialWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" rowspan="2">
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>串口数据 - 接收区</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTextEdit" name="edtReceive"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>串口数据 - 发送区</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTextEdit" name="edtSend">
           <property name="midLineWidth">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="2">
       <widget class="QCheckBox" name="chkSend">
        <property name="text">
         <string>16进制发送</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="4" column="1">
            <widget class="QComboBox" name="cmbStop">
             <property name="currentText">
              <string>1</string>
             </property>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="lblSwitch">
             <property name="text">
              <string>串口操作</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="lblCheckBit">
             <property name="text">
              <string>校验位</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="cmbCheck">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="currentText">
              <string>None</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lblDataBit">
             <property name="text">
              <string>数据位</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="btnSwitch">
             <property name="text">
              <string>打开串口</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="lblStopBit">
             <property name="text">
              <string>停止位：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cmbBit">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="currentText">
              <string>152000</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lblRate">
             <property name="text">
              <string>波特率</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="cmbDataBit">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
             <property name="currentText">
              <string>5</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lblSerial">
             <property name="text">
              <string>串口选择</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cmbSerial">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="chkRec">
        <property name="text">
         <string>16进制接收</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnClearSend">
        <property name="text">
         <string>清除发送</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnClearRec">
        <property name="text">
         <string>清除接收</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnSendData">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>100</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>发送数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSendMs">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>60</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>0</y>
        <width>90</width>
        <height>127</height>
       </rect>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkTimSend">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>定时发送</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblStatus">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>171</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
